### ISSUE TITLE
suggestion pass worker code as blob

### ISSUE THREAD

**Issue by shevernitskiy** [eliassjogreen/parry#1]
Did you investigate possibility to provide worker part as blob? Kinda:
```ts
new Worker(URL.createObjectURL(new Blob(workercode.toString()))
```
Potentially it could eliminate dependency on separate file for worker.

**Comment by eliassjogreen** [comment]
Yeah, made this a long time ago but that was originally how I wanted it however deno did not support that back then.

**Comment by eliassjogreen** [comment]
A contribution is welcome :D, otherwise I might only get to it far in the future :P.
